tests/test_assets/disease_correlation/output/disease_correlation.json

```diff
@@ -0,0 +1,13122 @@
+[
+  {
+    "model": "5xFAD",
```

Comment:
For simplicity's sake, do you need this many models/clusters/tissues for testing? Can you cut it down to like 2 model+age+sex combinations, and 2-3 clusters, and 1-2 tissues?